app/vmui/packages/vmui/src/components/TopQueries/TopQueryPanel/TopQueryPanel.tsx

```diff
@@ -0,0 +1,89 @@
+import React, {FC, useState} from "react";
+import Box from "@mui/material/Box";
+import {TopQuery} from "../../../types";
+import Typography from "@mui/material/Typography";
+import Tooltip from "@mui/material/Tooltip";
+import InfoIcon from "@mui/icons-material/Info";
+import Accordion from "@mui/material/Accordion";
+import AccordionSummary from "@mui/material/AccordionSummary";
+import ExpandMoreIcon from "@mui/icons-material/ExpandMore";
+import AccordionDetails from "@mui/material/AccordionDetails";
+import Tabs from "@mui/material/Tabs";
+import Tab from "@mui/material/Tab";
+import TableChartIcon from "@mui/icons-material/TableChart";
+import CodeIcon from "@mui/icons-material/Code";
+import TopQueryTable from "../TopQueryTable/TopQueryTable";
+import JsonView from "../../CustomPanel/Views/JsonView";
+
+interface TopQueryPanelProps {
+  rows: TopQuery[],
+  title: string,
+  description: string
+}
+const tabs = ["table", "JSON"];
+
+const TopQueryPanel: FC<TopQueryPanelProps> = ({rows, title, description}) => {
+
+  const [activeTab, setActiveTab] = useState(0);
+
+  return (
+    <Accordion
+      defaultExpanded={true}
+      sx={{
+        mt: 2,
+        border: "1px solid",
+        borderColor: "primary.light",
+        boxShadow: "none",
+        "&:before": {
+          opacity: 0
+        }
+      }}
+    >
+      <AccordionSummary
+        sx={{
+          p: 2,
+          bgcolor: "primary.light",
+          minHeight: "64px",
+          ".MuiAccordionSummary-content": { display: "flex", alignItems: "center" },
+        }}
+        expandIcon={<ExpandMoreIcon />}
+      >
+        <Tooltip arrow title={description}>
+          <InfoIcon color="info" sx={{mr: 1}}/>
+        </Tooltip>
+        <Typography variant="h6" component="h6">
+          {title}
+        </Typography>
+      </AccordionSummary>
+      <AccordionDetails sx={{p: 0}}>
+        <Box width={"100%"}>
+          <Box sx={{ borderBottom: 1, borderColor: "divider" }}>
+            <Tabs
+              value={activeTab}
+              onChange={(e, val) => setActiveTab(val)}
```

Comment:
nit: we can use one arrow function instead of call callback in the render part
```javascript
const handleChange = (e, val) => setActiveTab(val)
```
than 
```javascrip
onChange={handleChnage}
```